### PR TITLE
chore(md): add chinese doc

### PR DESCRIPTION
**Pull request by ShanaMaid**

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+[![Build Status](https://travis-ci.org/murlokswarm/app.svg?branch=master)](https://travis-ci.org/murlokswarm/app)
+[![Go Report Card](https://goreportcard.com/badge/github.com/murlokswarm/app)](https://goreportcard.com/report/github.com/murlokswarm/app)
+[![Coverage Status](https://coveralls.io/repos/github/murlokswarm/app/badge.svg?branch=master)](https://coveralls.io/github/murlokswarm/app?branch=master)
+[![awesome-go](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/avelino/awesome-go#readme)
+[![GoDoc](https://godoc.org/github.com/murlokswarm/app?status.svg)](https://godoc.org/github.com/murlokswarm/app)
+
+一个使用
+[Go](https://golang.org), [HTML](https://en.wikipedia.org/wiki/HTML5),
+[CSS](https://en.wikipedia.org/wiki/Cascading_Style_Sheets)构建的多平台UI框架
```
Comment (maxence-charriere) [inline README-CN.md]: Looks like there is an encoding problem here.
screen shot 2018-09-12 at 12 03 48 am

File: README-CN.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+|Linux|✖|
+
+<a name="hello"></a>
+
+## Hello world
+
+### 创建步骤
+
+```sh
+# Go to your repository:
```
Comment (maxence-charriere) [inline README-CN.md]: Should this be translated?

File: README-CN.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+cd YOUR_REPO
+
+# Init the repo:
```
Comment (maxence-charriere) [inline README-CN.md]: Should this be translated?

File: README-CN.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+goapp mac init
+```
+
+### 代码
+
+```go
+// YOUR_REPO/main.go
```
Comment (maxence-charriere) [inline README-CN.md]: Should this be translated?
你的回购/main.go ?

File: README-CN.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+    </h1>
+    <input value="{{.Name}}" placeholder="Write a name..." onchange="Name" autofocus>
+</div>
+    `
+}
+
+func main() {
+    app.Import(&Hello{})
+
+    // Use mac driver with Hello compo.
```
Comment (maxence-charriere) [inline README-CN.md]: Should this be translated?

File: README-CN.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+    app.Run(&mac.Driver{
+        URL: "/hello",
+    })
+}
+```
+
+### 构建并运行
+
+```sh
+# Build and run with debug mode:
```
Comment (maxence-charriere) [inline README-CN.md]: Should this be translated?

File: README-CN.md [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+
+* [Notifications](https://godoc.org/github.com/murlokswarm/app#NewNotification)
+* [FilePanel](https://godoc.org/github.com/murlokswarm/app#NewFilePanel)
+* [SaveFilePanel](https://godoc.org/github.com/murlokswarm/app#NewSaveFilePanel)
+* [Share](https://godoc.org/github.com/murlokswarm/app#NewShare)
+
+### 组件
+
+[compo](https://godoc.org/github.com/murlokswarm/app#Compo) 代表一个可以独立、可复用的UI组件. 它暴露的UI的HTML可以通过Go的一些基础库中提供的模板语法去进行自定义。
+[模板语法](https://golang.org/pkg/text/template/) defined in the Go
```
Comment (maxence-charriere) [inline README-CN.md]: Should this be translated?

File: README-CN.md [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+
+### 驱动
+
+[driver](https://godoc.org/github.com/murlokswarm/app#Driver) 代表app后台的具体运行方式。它暴露一些`Go`的操作方法去创建/修改UI和调用它们，并且会针对于特定于平台进行实现。
+<a name="goapp"></a>
+
+## 官方cli工具:Goapp
+
+Goapp是一个用来构建和运行通过`app`生成pakage的应用的官方命令行工具。
+Goapp is a CLI tool to build and run apps built with the app package.
```
Comment (maxence-charriere) [inline README-CN.md]: Should this be translated?

File: README-CN.md [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+
+<a name="architecture"></a>
+
+## 架构
+
+![ui architecture](https://github.com/murlokswarm/app/wiki/assets/architecture.png)
+
+### 元素
+
+一个 [Elem](https://godoc.org/github.com/murlokswarm/app#Elem) 代表一个UI组件. 部分元素可以
```
Comment (maxence-charriere) [inline README-CN.md]: Look like there is encoding issue here:
screen shot 2018-09-12 at 12 12 24 am

File: README-CN.md [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+### 组件
+
+[compo](https://godoc.org/github.com/murlokswarm/app#Compo) 代表一个可以独立、可复用的UI组件. 它暴露的UI的HTML可以通过Go的一些基础库中提供的模板语法去进行自定义。
+[模板语法](https://golang.org/pkg/text/template/) defined in the Go
+standard library. 组件能够在
+[elems](https://godoc.org/github.com/murlokswarm/app#ElemWithCompo) 里使用并且支持HTML自定义化。
+
+### 驱动
+
+[driver](https://godoc.org/github.com/murlokswarm/app#Driver) 代表app后台的具体运行方式。它暴露一些`Go`的操作方法去创建/修改UI和调用它们，并且会针对于特定于平台进行实现。
```
Comment (maxence-charriere) [inline README-CN.md]: Encoding issue:
screen shot 2018-09-12 at 12 13 26 am

**Comment by ShanaMaid** [comment]
ok, i wil fix problems in free time,

File: README-CN.md (hunk 1 above)
Comment (ShanaMaid) [reply README-CN.md]: done

File: README-CN.md (hunk 2 above)
Comment (ShanaMaid) [reply README-CN.md]: done

File: README-CN.md (hunk 3 above)
Comment (ShanaMaid) [reply README-CN.md]: done

File: README-CN.md (hunk 4 above)
Comment (ShanaMaid) [reply README-CN.md]: done

File: README-CN.md (hunk 5 above)
Comment (ShanaMaid) [reply README-CN.md]: done

File: README-CN.md (hunk 6 above)
Comment (ShanaMaid) [reply README-CN.md]: done

File: README-CN.md (hunk 7 above)
Comment (ShanaMaid) [reply README-CN.md]: done

File: README-CN.md (hunk 8 above)
Comment (ShanaMaid) [reply README-CN.md]: done

File: README-CN.md (hunk 9 above)
Comment (ShanaMaid) [reply README-CN.md]: done

File: README-CN.md (hunk 10 above)
Comment (ShanaMaid) [reply README-CN.md]: done

**Comment by ShanaMaid** [comment]
for Translation, you can put it bottom.

File: README.md [hunk 11]
```diff
@@ -12,6 +12,8 @@ A multi-platform UI framework that uses
 
 ![ui demo](https://github.com/murlokswarm/app/wiki/assets/ui-demo-large.gif)
 
+English | [中文](./README-CN.md)
```
[the remark this answers is not in the document]
Comment (ShanaMaid) [reply README.md]: done

File: README-CN.md [hunk 12]
```diff
@@ -0,0 +1,201 @@
+# app
```
[the remark this answers is not in the document]
Comment (ShanaMaid) [reply README-CN.md]: done

**Comment by ShanaMaid** [comment]
ok, all is done!
please review once again?

**Comment by maxence-charriere** [comment]
Thanks for this :).

**Pull request merged**